ledger-core/network/nds/uniproto/l1/api_transport.go

```diff
@@ -37,7 +37,8 @@ type VerifyPeerCertificateFunc func(rawCerts [][]byte, verifiedChains [][]*x509.
 type OutTransportFactory interface {
 	// ConnectTo resolves provided address and makes a connection
 	// Can return (nil, nil) when the given address is of unsupported type.
-	ConnectTo(nwapi.Address) (OutTransport, error)
+	ConnectTo(nwapi.Address) (OneWayTransport, error)
+	// todo: ConnectToAndVerify(nwapi.Address, VerifyPeerCertificateFunc) (OneWayTransport, error)
```

Comment:
why do you need it here? 
VerifyPeerCertificateFunc only applies to TLS while OutTransportFactory is generic
please remove the comment